misc/extension_api_validation/4.0-stable_4.1-stable.expected

```diff
@@ -3,6 +3,38 @@ when run against the extension_api.json of the 4.0-stable tag (first part of the
 
 ## Changes between 4.0-stable and 4.1-stable
 
+GH-78289
+--------
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_16', from 4CAD1009 to B6D9EDE0. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_32', from 4CAD1009 to B6D9EDE0. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_64', from 4CAD1009 to B6D9EDE0. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_8', from 4CAD1009 to B6D9EDE0. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_buffer', from B11D7DBE to 06CC13A1. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_csv_line', from 84319684 to 971BFC8B. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_double', from 1647D661 to 13B5FAA6. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_float', from 1647D661 to 13B5FAA6. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_line', from 04FD3184 to 8A854628. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_pascal_string', from 04FD3184 to 8A854628. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_real', from 1647D661 to 13B5FAA6. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_string', from 04FD3184 to 8A854628. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/FileAccess/methods/store_var', from 2C04CC12 to 06FEBB7D. This means that the function has changed and no compatibility function was provided.
```

Comment:
These hash ones shouldn't need to be here if the compatibility methods were setup correctly. Skimming them, they seem fine? If you remove the "hash changed" lines from this file does CI pass or fail?